MapboxCoreNavigation/URLSession.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import MapboxDirections
+
+extension URLSession {
+    /// The user agent string for any HTTP requests performed directly within this library.
+    static let userAgent: String = {
```

Comment:
This method was copied wholesale from MapboxNavigation. We should either remove the version in MapboxNavigation in favor of this one or have the version in MapboxNavigation call this version and insert the MapboxNavigation bundle version.